primitives/primproc/archcheck.h

```diff
@@ -0,0 +1,57 @@
+/* Copyright (C) 2014 InfiniDB, Inc.
```

Comment:
it's 2022 now, and Copyright (C) 2022 MariaDB Corporation

Reply:
Sorry, my bad. Just copied it from another file and didn't look.